scripts/content-utils/card-tree.ts

```diff
@@ -21,72 +21,135 @@ export type CardLeaf = {
     modifiers?: GameWorldModifier | GameWorldModifier[]
 }
 
+// TODO: Refactor CardTree to use this runtime type guard
+export function isCardTree(node: any): node is CardTree {
```

Comment:
Pleas avoid `any`. This is not a proper typeguard unless you specify which types you expect to receive. In this case I am guessing that the type should be `CardTree | CardLeaf | CardSequence`

Reply:
```
export function isCardTree(node: CardTree | CardLeaf): node is CardTree {
    return "card" in node
}
```